content/pt/case-studies/chinaunicom/index.html

```diff
@@ -0,0 +1,96 @@
+---
+title: Estudo de caso China Unicom
+
+linkTitle: chinaunicom
+case_study_styles: true
+cid: caseStudies
+css: /css/style_case_studies.css
+logo: chinaunicom_featured_logo.png
+featured: true
+weight: 1
+quote: >
+   O Kubernetes melhorou nossa experiência usando a infraestrutura de nuvem. Atualmente, não há tecnologia alternativa que possa substituí-lo.
+---
+
+<div class="banner1" style="background-image: url('/images/CaseStudy_chinaunicom_banner1.jpg')">
+  <h1> ESTUDO DE CASO:<img src="/images/chinaunicom_logo.png" class="header_logo" style="width:25%;margin-bottom:-1%"><br> <div class="subhead" style="margin-top:1%;line-height:1.4em">China Unicom: Como o Kubernetes alavancou a utilização de recursos, aumentando sua eficiência operacional<br>e menores custos em TI
```

Comment:
You don't need the `<br>` here -> `aumentando sua eficiência operacional<br>e menores custos em TI`

Reply:
Hi Felipe, I have maintained the same HTML element as the original, I think this is necessary because this is a special text like a magazine text that needs attention or a subtitle?

Reply:
@jcjesus Actually this is scaping another line. You can check your code here: https://deploy-preview-15347--kubernetes-io-master-staging.netlify.com/pt/case-studies/chinaunicom/

Reply:
Yes, equals than original,it's ok for me, and you?

https://kubernetes.io/case-studies/chinaunicom/

Reply:
Actually, you are using 3 lines because of the `<br>` :)

the original has 2

Reply:
Ok, I will remove, but in the original is the same behavior then you ask me, have a <br> element, and if you decrease the browser's window you can see in the original 3 lines too. I keep the same HTML elements then original.

Reply:
Ok, I will remove, but in the original is the same behaviour then you ask me, have an
element, and if you decrease the browser's window you can see in the original 3 lines too. I keep the same HTML elements then original.